### PR TITLE
Fix: roomMemberTest off-by-one error

### PR DESCRIPTION
**Pull request by Yoric**
Also, a few changes that should make the test slightly more readable.

### CODE REVIEW

**Comment by Gnuxie** [comment]
Buildkite is failing because of the linter btw, that should be fixed (stating in case it's ignored as part of ungoing CI rework)

File: test/integration/roomMembersTest.ts [hunk 1]
```diff
@@ -633,12 +643,28 @@ describe("Test: Testing RoomMemberManager", function() {
         ]) {
             experiment.addTo(EXPERIMENTS);
         }
+
+        // Sanity checks, before starting.
```
[the remark this answers is not in the document]
Comment (Yoric) [reply test/integration/roomMembersTest.ts:647]: You are right, thanks for the spot.

**Comment by Yoric** [comment]
Asking for a re-review to confirm that the new version is (a bit) easier to understand, at least wrt "witness" (now renamed "control") rooms.

File: test/integration/roomMembersTest.ts [hunk 2]
```diff
@@ -401,25 +401,37 @@ describe("Test: Testing RoomMemberManager", function() {
...
+        //     (we're not calling `!mjolnir rooms add` for this room), so none
+        //     of the operations of `!mjolnir since` shoud affect it. We are
+        //     using it to control, at the end of each experiment, that none of
+        //     the `!mjolnir since` operations affect it.
+        // - room 1, also known as the "control protected room" is protected
+        //     (we are calling `!mjolnir rooms add` for this room), but we are
+        //     never directly requesting any `!mjolnir since` action against
+        //     this room. We are using it to control, at the end of each experiment,
+        //     that none of the `!mjolnir since` operations that should target
+        //     one single other room also affect that room. It is, however, affected
```
Comment (Gnuxie) [inline test/integration/roomMembersTest.ts:416]: Makes sense

**Review by Gnuxie — APPROVED** [review]

**Pull request merged**